### PR TITLE
Update to France

### PR DESCRIPTION
**Pull request by p-heckler**
Hi, 
I have updated the French file, added and edited some courts, and corrected some minor typos! I hope the formatting is valid, there are a few aspects of which I am unsure. I will probably also review EU and International files to add French translations when applicable, when I have the time for it!

### CODE REVIEW

File: src/juris-fr-desc.json [hunk 1]
```diff
@@ -1,64 +1,99 @@
...
-    },
     "cass": {
-      "name": "Cour de Cassation"
+      "name": "Cour de cassation",
+	"abbrev": "Cass."	    
     },
-    "cass~civ1": {
-      "name": "Cour de Cassation|1re chambre civile"
+    "cass~civ.1": {
+      "name": "< 1re Chambre civile"
```
Comment (georgd) [inline src/juris-fr-desc.json:8]: A comma is missing here and between many other "name": "..." and "abbrev": "..."

File: src/juris-fr-desc.json [hunk 2]
```diff
@@ -1,64 +1,99 @@
...
+	    "abbrev": "< civ. 1re"
     },
-    "cass~civ2": {
-      "name": "Cour de Cassation|2e chambre civile"
+    "cass~civ.2": {
```
Comment (georgd) [inline src/juris-fr-desc.json:11]: The period in `civ.2` mit lead to errors. It’s just a code so no need to make it look nicer :)

File: src/juris-fr-desc.json [hunk 3]
```diff
@@ -1,64 +1,99 @@
...
+      "name": "< Assemblée plénière"
+	    "abbrev": "Ass. plen."
     },
+	  "cass~mixte": {
+		  "name": "< Chambre mixte"
+		  "abbrev": "< ch. mixte"
+
     "cc": {
-      "name": "Conseil Constitutionelle"
+      "name": "Conseil constitutionel"
```
Comment (georgd) [inline src/juris-fr-desc.json:40]: Missing 'n' in `constitutionel`

**Comment by georgd** [comment]
Great to see these improvements for French abbreviations!

File: src/juris-fr-desc.json (hunk 2 above)
Comment (p-heckler) [reply src/juris-fr-desc.json:11]: I noticed the ECJ followed this format, I wasn't sure whether or not it was standard!

**Comment by p-heckler** [comment]
Thank you @georgd for the review and sorry for the basic JSON errors, not quite used to writing in the format!

File: src/juris-fr-desc.json [hunk 4]
```diff
@@ -1,64 +1,99 @@
...
+	    "abbrev": "soc."
     },
+    "cass~plen": {
+      "name": "< Assemblée plénière",
+	    "abbrev": "Ass. plen."
+    },
+	  "cass~mixte": {
+		  "name": "< Chambre mixte",
+		  "abbrev": "< ch. mixte"
+
```
Comment (georgd) [inline src/juris-fr-desc.json:38]: I missed that before: the closing brace `},` is missing here.

File: src/juris-fr-desc.json [hunk 5]
```diff
@@ -1,64 +1,99 @@
...
+		  "abbrev": "C.A. >"
+	  }, 
+	  "tsa": {
+		  "name": "Tribunal supérieur d'appel",
+		  "abbrev": "T.S.A."
+	  },
+	  "caa": {
+	  "name": "Cour administrative d'appel",
+		  "abbrev": "C.A.A. >"
+	  },
```
Comment (georgd) [inline src/juris-fr-desc.json:72]: This comma in line 72 is illegal as well.

**Comment by georgd** [comment]
> Thank you @georgd for the review and sorry for the basic JSON errors, not quite used to writing in the format!

You’re welcome. For working with this kind of files, a validating editor is incredibly helpful, like Atom editor or Visual Studio Code (or Vim with certain plugins ;))

File: src/juris-fr-desc.json (hunk 2 above)
Comment (georgd) [reply src/juris-fr-desc.json:11]: Right, I overlooked that.

**Comment by fbennett** [comment]
The format of the ``desc`` files changed in the interim, so I have downloaded the file as revised by @p-heckler, converted it to the new format, and pushed it to the repo at https://github.com/Juris-M/legal-resource-registry/commit/c97d87c5094aba783b9a46dbea1261ecb6d82adb

**Pull request closed without merging**